test/utils/utils.go

```diff
@@ -121,6 +121,7 @@ func (p *PodmanTest) WaitForContainer() bool {
 		}
 		time.Sleep(1 * time.Second)
 	}
+	fmt.Printf("WaitForContainer(): timed out\n")
```

Comment:
This would've been really helpful when I was debugging the `PodmanPID` bug above.